### PR TITLE
fix(chapters): fix chapters grammar

### PR DESCRIPTION
**Pull request by pegak**

### CODE REVIEW

File: chapters/qa-1.mdx [hunk 1]
```diff
@@ -18,13 +18,13 @@ Pojďme ale postupně - HTML (HyperText markup language) patří do skupiny [Zna
...
 
 ![Struktura DOM](/images/chapter/qa-1-1.png "Jednoduchá ukázka struktury DOM")
 
 ## CSS
 
-HTML prohlížeči říká, co chceme zobrazit (nadpis, obrázek), ale neříká, jak to chceme zobrazit (barva, odsazení od okolí, barva pozadí atd.). Pro úpravu vzhledu se používají [Kaskádové styly - CSS](https://www.itnetwork.cz/html-css/webove-stranky/jak-psat-moderni-web-html-tutorial-uvod-do-css) (Cascading Style Sheet). Pomocí CSS můžeme definovat celý vzhled dokumentu, mimojiné třeba barvu textu a pozadí, fontebo také prostor mezi HTML prvky. Jednoduché CSS vypadá takto:
+HTML prohlížeči říká, co chceme zobrazit (nadpis, obrázek), ale neříká, jak to chceme zobrazit (barva, odsazení od okolí, barva pozadí atd.). Pro úpravu vzhledu se používají [Kaskádové styly - CSS](https://www.itnetwork.cz/html-css/webove-stranky/jak-psat-moderni-web-html-tutorial-uvod-do-css) (Cascading Style Sheet). Pomocí CSS můžeme definovat celý vzhled dokumentu, mimojiné třeba barvu textu a pozadí, fontebo také prostor mezi HTML prvky. Jednoduché CSS vypadá takto:
```
Comment (xhemalov) [inline chapters/qa-1.mdx]: fontebo x font nebo?

File: chapters/qa-1.mdx [hunk 2]
```diff
@@ -74,21 +74,21 @@ Možná struktura je tato:
...
 
 ---
 
-Neboj se si svým blogem pohrát. Nabereš tak zkušenosti, které zúročíš v pozdějších částech kurzu. V kapitole osm zjistíš, jak složku nahrát na Github (jak to máme my), kde se na to může podívat třeba i tvůj možný budoucí zaměstnavatel. Prozatím si ji můžeš uložit v nějakém cloudu jako je Google Drive, One Drive nebo Dropbox. Pokud si nebudeš vědět rady, klidně se na nás obrať na našem [Discordu](https://discord.gg/QbYswwYUPU).
+Neboj se si svým blogem pohrát. Nabereš tak zkušenosti, které zúročíš v pozdějších částech kurzu. V kapitole osm zjistíš, jak složku nahrát na Github (jak to máme my), kde se na to může podívat třeba i tvůj možný budoucí zaměstnavatel. Prozatím si ji můžeš uložit v nějakém cloudu, jako je Google Drive, One Drive nebo Dropbox. Pokud si nebudeš vědět rady, klidně se na nás obrať na našem [Discordu](https://discord.gg/QbYswwYUPU).
```
Comment (xhemalov) [inline chapters/qa-1.mdx]: Can we add a link to the Github repository?

**Review by Mmykron — APPROVED** [review]

File: chapters/qa-4.mdx [hunk 3]
```diff
@@ -200,11 +200,11 @@ let osoba = {
 }
 ```
 
-Pokud chceme zjistit věk naší osoby, musíme to udělat přes tečkovou notaci a napsat osoba.age. Pokud chceme zjistit jestli je osoba plnoletá, použijeme také tečkovou notaci, ale protože je to funkce přidáme závorky (musíme funkci zavolat - to platí u jakékoliv funkce, nikoliv těch, které patří do nějakého objektu) => osoba.isAdult().
+Pokud chceme zjistit věk naší osoby, musíme to udělat přes tečkovou notaci a napsat osoba.age. Pokud chceme zjistit, jestli je osoba plnoletá, použijeme také tečkovou notaci, ale protože je to funkce, přidáme závorky (musíme funkci zavolat - to platí u jakékoliv funkce, nikoliv těch, které patří do nějakého objektu) => osoba.isAdult().
```
Comment (xhemalov) [inline chapters/qa-4.mdx:203]: I do not understand the meaning of the text in ()

File: chapters/qa-5.mdx [hunk 4]
```diff
@@ -1,69 +1,69 @@
...
+Podívej na video od [Czechitas](https://www.czechitas.cz/lekce-online-kurzu/javascript-lekce-2) a článek pod ním. Vše, co je ve videu zmíněno, lze dělat přímo z konzole prohlížeče, podobně jako jsi pracoval s Javascriptem v předchozí kapitolách. Kód se ukládá do samostatných souborů, pokud s ním chceme dále pracovat, mít ho funkční i po obnovení stránky či sdílet s kolegy.
 
 Všechny příkazy, uvedené v materiálech výše, můžeš používat v konzoli v prohlížeči a měnit si stránky pod rukama. Jdi na náš [cestovatelský blog](https://nauc-me-it.github.io/ukazkovy-blog/index.html), otevři si konzoli a zkus:
 
 -   Přepsat titulek stránky (to, co vidíš nahoře v prohlížeči - název záložky)
 -   Přepsat hlavní nadpis v hlavičce stránky
 -   Změň barvu všech odkazů, které jsou podkategorií (např. Dánsko či Švédsko)
 
-![Detail na podmenu k úkolu](/images/chapter/qa-5-1.png "Obrázek z blogu, kde je zvýrazněno, jakou část chceme abys změnil.")
+![Detail na podmenu k úkolu](/images/chapter/qa-5-1.png "Obrázek z blogu, kde je zvýrazněno, jakou část chceme abys změnil.")
```
Comment (xhemalov) [inline chapters/qa-5.mdx:26]: I do not understand what podmenu means.

File: chapters/qa-6.mdx [hunk 5]
```diff
@@ -3,15 +3,15 @@ title: Síť
...
+Více o tom, jak funguje internet a i něco o jeho historii se můžeš dozvědět na webu [Jak na Internet](https://www.jaknainternet.cz/), který provozuje správce české [domény](https://www.youtube.com/watch?v=J44jvmoDxzo) [CZ.NIC](https://www.nic.cz/). Doporučujeme hlavně sérii [Správa a fungování internetu](https://www.jaknainternet.cz/page/1790/bezpecnostni-tymy/), zajímavá a přínosná jsou i další jejich videa. Pod videem se nacházejí články, které obsahují další informace o daném tématu. V sérii se můžeš dozvědět o doménách prvního řádu, kterými můžou být například [národní i generické](https://eprehledy.cz/narodni-domeny-statu.php) domény. Náš web naucme.it používá italskou národní doménu. Vlastníme i doménu naucmeit.cz, který přesměrovává na naucme.it. Je běžné mít více domén - například, aby se někdo netvářil jako ty a nepodváděl uživatele - krásným příkladem je [gogle.com](http://gogle.com) - přesměruje tě na hlavní doménu [google.com](https://google.com).
```
Comment (xhemalov) [inline chapters/qa-6.mdx:10]: naucmeit.cz, která?

File: chapters/qa-6.mdx [hunk 6]
```diff
@@ -3,15 +3,15 @@ title: Síť
...
+Zaměříme se na protokoly sedmé aplikační vrstvy. Protokoly, se kterými pracuje prohlížeč. [Protokolů](https://akela.mendelu.cz/~jprich/vyuka/os/unix/telnet.html) je celá řada. Při práci na webu se nejčastěji setkáš s protokoly [HTTPS ](https://www.wikiwand.com/cs/HTTPS) a [HTTP](https://www.wikiwand.com/cs/Hypertext_Transfer_Protocol). Více o HTTP nalezneš na [Jakpsatweb.cz](https://www.jakpsatweb.cz/server/http-protokol.html), což je sice starší stránka, avšak stále obsahuje spoustu hodnotných informací. Protokoly HTTP a HTTPS jsou téměř stejné, nicméně HTTPS je zabezpečené. To znamená, že případný útočník si nemůže zobrazit obsah, který se posílá. Dříve byl tento protokol využívaný při výměně citlivých dat. Jistě bys nechtěl, aby se heslo do tvého internetového bankovnictví posílalo bez jakéhokoliv zabezpečení. Nyní se stále více tlačí na to, aby se se používal všude a některé vyhledávače (např. Google) penalizují stránky, které toto zabezpečení nepoužívají. Že je stránka zabezpečená, poznáš díky zámečku vedle URL adresy webu, na kterém se nacházíš. Nebo podle https: na začátku URL.
```
Comment (xhemalov) [inline chapters/qa-6.mdx:14]: Is meaningful the space in [HTTPS ]?

File: chapters/qa-6.mdx [hunk 7]
```diff
@@ -37,17 +37,17 @@ Občas nastane chyba - pokusíš se změnit oběd na příští týden příliš
...
+Ty důležité jsou popsány na [Strafelda.cz - Co je stavový kód](https://www.strafelda.cz/stavovy-kod), pokud by ses setkal s nějakým zde neuvedeným můžeš využít i [Wikipedii - Stavové kódy](https://www.wikiwand.com/cs/Stavov%C3%A9_k%C3%B3dy_HTTP).
 
 ## DevTools - záložka Network
 
-V DevTools můžeš zjistit mnoho informací o požadavku například jeho výsledek a stavový kód. Prozkoumáme cvičný web [https://astronauts.vercel.app/](https://astronauts.vercel.app/). Abys dosáhl stejného výsledku jako my na obrázcích je potřeba drobná kosmetická úprava pomocí DevTools. Změň u tagu _img_ atribut _src_ z _rocket.png_ na _rocket2.png_. Poté přejdi do záložky **Network**.
+V DevTools můžeš zjistit mnoho informací o požadavku například jeho výsledek a stavový kód. Prozkoumáme cvičný web [https://astronauts.vercel.app/](https://astronauts.vercel.app/). Abys dosáhl stejného výsledku jako my na obrázcích je potřeba drobná kosmetická úprava pomocí DevTools. Změň u tagu _img_ atribut _src_ z *rocket.png* na _rocket2.png_. Poté přejdi do záložky **Network**.
```
Comment (xhemalov) [inline chapters/qa-6.mdx:46]: why does the rocket.png have a different emphasis than the rocket2.png?

**Review by xhemalov — CHANGES_REQUESTED** [review]

File: chapters/qa-1.mdx (hunk 1 above)
Comment (pegak) [reply chapters/qa-1.mdx]: Done.

File: chapters/qa-1.mdx (hunk 2 above)
Comment (pegak) [reply chapters/qa-1.mdx]: Done.

File: chapters/qa-4.mdx (hunk 3 above)
Comment (pegak) [reply chapters/qa-4.mdx:203]: Improved.

File: chapters/qa-6.mdx (hunk 5 above)
Comment (pegak) [reply chapters/qa-6.mdx:10]: Fixed.

File: chapters/qa-6.mdx (hunk 6 above)
Comment (pegak) [reply chapters/qa-6.mdx:14]: Done.

File: chapters/qa-6.mdx (hunk 7 above)
Comment (pegak) [reply chapters/qa-6.mdx:46]: It is enforced automatically, but nice catch!

**Review by xhemalov — APPROVED** [review]

**Pull request merged**